invenio_records_resources/services/uow.py

```diff
@@ -231,6 +233,18 @@ def on_post_commit(self, uow):
         )
 
 
+class NotificationOp(Operation):
+    """A notification operation."""
+
+    def __init__(self, notification):
+        """Initialize operation."""
+        self._notification = notification
+
+    def on_post_commit(self, uow):
+        """Start task to send notification."""
+        broadcast_notification.delay(self._notification)
```

Comment:
I am sure Lars will suggest to change `broadcast_notification` to `broadcast` only! :) :D